sql/core/src/test/scala/org/apache/spark/sql/execution/exchange/EnsureRequirementsSuite.scala

```diff
@@ -133,4 +134,27 @@ class EnsureRequirementsSuite extends SharedSparkSession {
       }.size == 2)
     }
   }
+
+  test("SPARK-35989: Do not remove REPARTITION_BY_NUM shuffle if AQE is enabled") {
+    import testImplicits._
+    Seq(true, false).foreach { enableAqe =>
+      withSQLConf(SQLConf.ADAPTIVE_EXECUTION_ENABLED.key -> enableAqe.toString,
+        SQLConf.SHUFFLE_PARTITIONS.key -> "3",
+        SQLConf.AUTO_BROADCASTJOIN_THRESHOLD.key -> "-1") {
```

Comment:
nit:

```suggestion
          SQLConf.SHUFFLE_PARTITIONS.key -> "3",
          SQLConf.AUTO_BROADCASTJOIN_THRESHOLD.key -> "-1") {
```

Reply:
Seems two Indentation here is right ?

Reply:
I usually follow method definition one here (https://github.com/databricks/scala-style-guide#spacing-and-indentation) because the parameters and body are inlined, and makes it a bit less readable. but im fine as is either.

Reply:
I see the reason, make sense